src/components/Admin/AddPopUpClinic/AdminPopUpClinic.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import { Card, Form, FormLayout, TextField } from "@shopify/polaris";
+import { useTranslation } from "react-i18next";
+import { postalCodeIsValid } from "../../../utils/postalCode";
+import { phoneNumberIsValid } from "../../../utils/phoneNumber";
+
+export interface IPopUpClinic {
+  name: string;
+  date: string;
+  street: string;
+  city: string;
+  province: string;
+  postalCode: string;
+  phone: string;
+  websiteUrl: string;
+  [key: string]: string;
+}
+
+const handleSubmit = () => {
+  return true;
+};
+
+export function AdminPopUpClinic() {
+  const { t } = useTranslation();
+
+  const [popUpClinic, setPopUpClinc] = useState<IPopUpClinic>({
```

Comment:
No action required, but I generally avoid objects for component state (and definitely avoid deeply nested objects due to immutability concerns...) The official guidance suggests splitting out state into individual pieces or pieces of state that change together: https://reactjs.org/docs/hooks-faq.html#should-i-use-one-or-many-state-variables.